VirtualEconomyFramework/VENFTApp-Blazor/Components/Account/CreateAccount.razor

```diff
@@ -89,28 +89,25 @@
                                 {
                                     if (passwordConfirm != password)
                                     {
-                                        <Alert Type="@AlertType.Warning"
-                                               Message="Warrning"
-                                               Description="Passwords not match!"
-                                               ShowIcon="true" />
+                                        <VENFTApp_Blazor.Components.Alert Type="VENFTApp_Blazor.Components.AlertType.Warning">
+                                            <Message>Passwords not match!</Message>
+                                        </VENFTApp_Blazor.Components.Alert>

```

Comment:
Message (now called Title) was repeated and sometimes there was a typo, like `Warrning`.
Title is now optional. Default title is type name.

Reply:
It looks cool. I fixed small bug with switched warning and success when creating the account.